test/integration-tests/Uniswap.test.ts

```diff
@@ -27,17 +28,24 @@ import { RoutePlanner, CommandType } from './shared/planner'
 import hre from 'hardhat'
 import { getPermitSignature, getPermitBatchSignature, PermitSingle } from './shared/protocolHelpers/permit2'
 import { encodePathExactInput, encodePathExactOutput } from './shared/swapRouter02Helpers'
+import getPermitNFTSignature from './shared/getPermitNFTSignature'
+import { FeeAmount } from '@uniswap/v3-sdk'
+import { encodeDecreaseLiquidity, encodeCollect, encodeBurn } from './shared/encodeCall'
 const { ethers } = hre
 
 describe('Uniswap V2 and V3 Tests:', () => {
   let alice: SignerWithAddress
   let bob: SignerWithAddress
+  let eve: SignerWithAddress
   let router: UniversalRouter
   let permit2: IPermit2
   let daiContract: Contract
   let wethContract: Contract
   let usdcContract: Contract
   let planner: RoutePlanner
+  let v3NFTPositionManager: INonfungiblePositionManager
+
+  let tokenId: BigNumber
```

Comment:
little bit unclear what this is in the grand scheme of _all_ UR tests. Maybe `positionIDv3`